bootstrap.sh

```diff
@@ -15,6 +15,8 @@ kubectl apply -f .infrastructure/app/nodeport.yml
 kubectl apply -f .infrastructure/app/hpa.yml
 kubectl apply -f .infrastructure/app/deployment.yml
 
+kubectl apply -f .infrastructure/security/rbac.yml
+
 # Install Ingress Controller
 kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
```

Comment:
The script installs the Ingress Controller using a URL. Ensure that the URL is correct and accessible. If there are any network restrictions, this might fail.